scripts/calc_functions_soil_modelling.R

```diff
@@ -0,0 +1,64 @@
+library(tidyr)
+
+get_monthly_Cinputs_animals <- function (animal_inputs, animal_factors, scenario_chosen, parcel){
+  animals = merge(x = filter(animal_inputs,scenario==scenario_chosen & parcel_ID==parcel), y = animal_factors, by = "species", all.x = TRUE) %>% 
+    mutate (C_inputs_manure_kg_per_ha_per_year= n_animals*c_kg_per_year_per_animal/area*grazing_days/365)
+  tC_inputs_per_ha_per_year = sum(animals$C_inputs_manure_kg_per_ha_per_year)*1e-3
+  return(tC_inputs_per_ha_per_year)
+}
+
+
+get_monthly_Cinputs_agroforestry <- function (agroforestry_inputs, agroforestry_factors, scenario_chosen, parcel){
+  zone=ifelse(lat_farmer<57,"Temperate","Boreal")
+  trees = merge(x = filter(agroforestry_inputs,scenario==scenario_chosen & parcel_ID==parcel), y = filter(agroforestry_factors,climatic_zone==zone), by = "tree_species", all.x = TRUE) %>% 
```

Comment:
I think some cleaning of this at some stage might be good - or perhaps just some additional line breaks to make it simpler to see everything.